src/open_inwoner/components/templates/components/List/ListItem.html

```diff
@@ -1,5 +1,5 @@
 {% load link_tags %}
-<li class="list-item{% if active %} list-item--active{% endif %}{% if compact %} list-item--compact{% endif %}">
+<li class="list-item{% if active %} list-item--active{% endif %}{% if compact %} list-item--compact{% endif %}{% if extra_compact %} list-item--extra-compact{% endif %}">
     {% if href %}<a class="link" href="{{ href }}">{% endif %}
     {% if caption %}<p class="p list-item__caption">{{ caption }}</p>{% endif %}
```

Comment:
Globally we need to give .list-item__caption a line-height of "--font-line-height-body-small" so that would be neat to add in this PR.